libs/agora/teams/src/lib/team-list/team-list.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="team-list">
   <div class="team-list-inner">
-    <ng-container *ngIf="_teams?.length">
+    @if (_teams.length) {
```

Comment:
converted a bunch of these *ngIf to the new syntax @if where possible.  @for will be tackled in the future